main.html.in

```diff
@@ -181,6 +181,9 @@
     </div>
   </div>
 
+  <div class="background-screen" id="background-screen">
```

Comment:
There is only ever one of these; does it really need both a class and an ID? The styles that apply to the class could just as easily apply to the ID.